src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,8 @@
+.catalog {
+  display: flex;
+  width: $catalog-width;
+  margin: 0 auto;
+  padding: 50px $catalog-padding 73px;
```

Comment:
Maybe side paddings is redundant. Block "catalog" horizontally centered because of "margin: 0 auto;". So if paddings will be removed block "catalog" will stay in the same position, just its width become smaller by 80px (two 40px paddings).

Reply:
Ok, about paddings that was just suggestion and you don't need to fix it